libs/api/domains/cms/src/lib/models/subpageHeader.model.ts

```diff
@@ -0,0 +1,32 @@
+import { Field, ObjectType } from '@nestjs/graphql'
+
+import { ISubpageHeader } from '../generated/contentfulTypes'
+import { Image, mapImage } from './image.model'
+
+@ObjectType()
+export class SubpageHeader {
+  @Field()
+  subpageId: string
+
+  @Field()
+  title: string
+
+  @Field()
+  summary: string
+
+  @Field({ nullable: true })
```

Comment:
I'm not sure nest graphQL can resolve the type here.
You might need to add a type function to resolve the type:
```suggestion
  @Field(() => Image, { nullable: true })
```